src/OXRS_WT32.h

```diff
@@ -18,9 +18,15 @@
 // REST API
 #define REST_API_PORT               80
 
+// Climate sensor update internal
+#define DEFAULT_CLIMATE_UPDATE_MS   60000L
+
 // Enum for the different connection states
 enum connectionState_t { CONNECTED_NONE, CONNECTED_IP, CONNECTED_MQTT };
 
+// callback to signal upstream climate values have changed
+typedef void (*climateUpdateCallback)(void);
```

Comment:
Would it make sense to add the temp/humidity values to the callback? Then you would only need the callback and not the extra `getClimate()` method. Or do you need to explicitly query the climate values some times?

Reply:
The settings screen collects data from different sources when triggered to update itself to ensure latest states from the owners, so query is needed.